AnotherSmallFile.txt

```diff
@@ -1,117 +0,0 @@
-Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-Phasellus porta dui non massa pellentesque semper.
-Nunc eget maximus metus.
-Donec laoreet aliquam leo ac tempus.
-Donec nulla ipsum, ultricies ac elit vel, dapibus fringilla neque.
-Proin sit amet consectetur velit.
-Morbi facilisis purus nec pretium mattis.
-Phasellus eu augue commodo, tincidunt nisl quis, porta nisi.
-
-Ut blandit vitae ligula ut auctor.
-Pellentesque nisl lectus, sagittis non semper quis, posuere quis tortor.
-Nullam bibendum aliquam luctus.
-Suspendisse pretium, magna sed dictum feugiat, augue lacus congue turpis, sit amet egestas ex nisi at leo.
-Sed pretium in arcu in tristique.
-Aenean aliquam mollis congue.
-Nam finibus turpis nibh, in consequat justo ullamcorper porttitor.
-Donec tempus arcu at arcu sodales, id euismod enim venenatis.
-Etiam sed arcu sapien.
-Sed tempor ex vel magna egestas posuere et eget orci.
-Sed sodales, lorem sed tempus lacinia, enim felis ultricies magna, ac auctor nulla orci sed magna.
-Integer ullamcorper venenatis ipsum, nec aliquam risus gravida in.
-Aenean tempus neque justo, in gravida purus facilisis ac.
-Vestibulum a elit fermentum, aliquam nisi ut, euismod dolor.
-Donec sit amet ultricies tortor, eget bibendum lectus.
-
-Maecenas laoreet nibh arcu, eu aliquam mauris feugiat non.
-Nam dignissim purus quis dolor tincidunt vulputate.
-Suspendisse aliquam iaculis commodo.
-Phasellus turpis ante, ornare sit amet ipsum vitae, volutpat ultrices justo.
-Pellentesque tincidunt posuere quam pretium consectetur.
-Etiam ac vehicula tortor, quis dapibus lectus.
-Ut ac efficitur elit.
-Morbi tempus eu mi a porta.
-
-Sed sem sem, molestie quis metus et, condimentum tincidunt diam.
-Donec efficitur lorem arcu, quis lobortis elit dictum vitae.
-In accumsan consequat mi, vel rhoncus sem tincidunt at.
-In aliquet, tellus ut blandit convallis, nunc nunc vulputate mauris, sed tristique augue orci sit amet turpis.
-Proin consequat eget urna et porta.
-Donec luctus rhoncus pulvinar.
-Mauris at dignissim mi.
```

Comment:
Reply from CodeFlow.

---
In reply to: [68511271](https://github.com/kivancmuslu/GitHubTest/pull/1#discussion_r68511271) [](ancestors = 68511271)